### PR TITLE
chore: udpate build instructions

### PR DESCRIPTION
**Pull request by IMPranshu**

### CODE REVIEW

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md [hunk 1]
```diff
@@ -44,44 +46,64 @@ yay -S curl base-devel sqlite openssl clang cmake ninja pkg-config gtk3 unzip
...
 * Run the setup script from the base directory
 ```bash
+cd appflowy
 ./frontend/scripts/install_dev_env/install_linux.sh
 ```
+![img.png](../../../../.gitbook/assets/build\_step\_two\_script.png)
+
+![img.png](../../../../.gitbook/assets/build\_step\_two\_warn\_success.png)
+
+`If you get the warning as shown above run following command`
```
Comment (annieappflowy) [inline essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md:73]: If you get the warning as shown above, run the following command:

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md [hunk 2]
```diff
@@ -99,15 +121,23 @@ cargo make -p production-linux-x86_64 appflowy-linux
...
-cd app_flowy/product/0.0.2/linux/Debug/AppFlowy
+cd app_flowy/product/0.0.5/linux/Debug/AppFlowy
 ```
 
 ```shell
 ./app_flowy
 ```
+![img.png](../../../../.gitbook/assets/build\_step\_four\_success.png)
+
+* A new window must appear running `AppFlowy` application as show below.
```
Comment (annieappflowy) [inline essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md:137]: A new window as shown below will show up after you run the application:

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md (hunk 1 above)
Comment (IMPranshu) [reply essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-linux.md:73]: Should I remove the "`"? or just the punctuation mark missing?

**Comment by IMPranshu** [comment]
closes AppFlowy-IO/AppFlowy#841

**Comment by annieappflowy** [comment]
Hi @IMPranshu  ,

Hope all is well with you. I'm writing to let you know we are relicensing this documentation repo from AGPL to CC BY-SA. You can find the context https://github.com/AppFlowy-IO/AppFlowy-Docs/issues/75.

You are receiving this because you’ve made significant contributions to this repo, and we are seeking consent from you to relicense your contributions, in order to adjust our licensing in the best interests of the community.

Is there a way I can send you a docusign? I'd need your name and email.
I can be found on our[ Discord ](https://discord.gg/9Q2xaN37tV)server.

Thanks,
Annie

**Comment by IMPranshu** [comment]
> Hi @IMPranshu  ,
> 
> Hope all is well with you. I'm writing to let you know we are relicensing this documentation repo from AGPL to CC BY-SA. You can find the context https://github.com/AppFlowy-IO/AppFlowy-Docs/issues/75.
> 
> You are receiving this because you’ve made significant contributions to this repo, and we are seeking consent from you to relicense your contributions, in order to adjust our licensing in the best interests of the community.
> 
> Is there a way I can send you a docusign? I'd need your name and email.
> I can be found on our[ Discord ](https://discord.gg/9Q2xaN37tV)server.
> 
> Thanks,
> Annie

Pranshu Kumar Agrawal
pranshukumar1999@gmail.com

**Comment by annieappflowy** [comment]
@IMPranshu , just sent you a DocuSign. Thanks for your support!

**Pull request merged**